content/references/data-api.md

```diff
@@ -2641,7 +2641,7 @@ Response:
 ## Get Validator
 [[Source]<br>](https://github.com/ripple/rippled-historical-database/blob/develop/api/routes/network/getValidators.js "Source")
 
-Get details of a single validator in the [consensus network](consensus.html). _(New in [v2.2.0][])_
+Get details of a single validator in the [consensus network](consensus.html). _(Updated in [v2.4.0][])_
```

Comment:
Do we want to provide a Try It button for this method?

Reply:
Now that the tool is updated, yes.